meson.build

```diff
@@ -2,7 +2,10 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-project('opentitan', 'c', 'cpp', version: '0.1',)
+project('opentitan', 'c', 'cpp', version: '0.1',
+        default_options : ['c_std=gnu99', 'cpp_std=c++14',
+                           'warning_level=1', 'werror=true',
+                           'debug=true',])
```

Comment:
Also, I just noticed this... can you format it to be
```
project(
  'opentitan', 'c', 'cpp',
  version: '0.1',
  default_options: [
    # ...
  ],
)
```

Sorry for missing this. It's kind of annoying that Meson doesn't have a formater and that their code examples are super style-inconsistent.